write_to_score.py

```diff
@@ -31,29 +33,76 @@ def write_tsvs_to_scores(
         The path to annotations and MuseScore3 scores, whose sub-directories and file names
         are aligned with those in the output TSV directory.
 
-    fuzzy : bool
```

Comment:
This now causes an error in the script. Please also remove the --fuzzy arg and its inclusion in the call to this function.

Reply:
[3c57d40](https://github.com/apmcleod/harmonic-inference/pull/73/commits/3c57d40ef6dbe84a6ddf85c34254e06756f872f2)